sql/core/src/main/scala/org/apache/spark/sql/execution/python/AggregateInPandasExec.scala

```diff
@@ -162,7 +162,10 @@ case class AggregateInPandasExec(
         argOffsets,
         aggInputSchema,
         sessionLocalTimeZone,
-        pythonRunnerConf).compute(projectedRowIter, context.partitionId(), context)
+        pythonRunnerConf,
+        pythonDataSent,
+        pythonDataReceived,
+        pythonNumRowsReceived).compute(projectedRowIter, context.partitionId(), context)
```

Comment:
Should we maybe just pass `metrics: Map[String, SQLMetric]` instead of each individually? Otherwise we will have to add new parameter here when we add new matrics.

Reply:
+1